fighters/rosetta/src/acmd/aerials.rs

```diff
@@ -126,6 +126,7 @@ unsafe fn rosetta_attack_air_hi_game(fighter: &mut L2CAgentBase) {
     }
     frame(lua_state, 45.0);
     if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 4.0);
```

Comment:
This motion rate should be set earlier in the move with a smaller multiplier to avoid strange animation inconsistencies